calliope/test/test_cli.py

```diff
@@ -31,7 +31,13 @@ def test_run_from_yaml(self):
 
         with runner.isolated_filesystem() as tempdir:
             result = runner.invoke(
-                cli.run, [_MODEL_NATIONAL, "--save_netcdf=output.nc"]
+                cli.run, [
```

Comment:
I will deal with this in another PR (or directly on master) since there are other files in need of Black's magic touch and I don't want to muddy the waters with lots of files changed.